core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/FailureModes.scala

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.dsde.rawls.monitor.migration
+
+object FailureModes {
```

Comment:
I'll remove this entirely in the next bucket migration PR -- there is a duplicated set of `MultiregionalBucketMigrationFailureModes` that I neglected to actually switch over to